include/nbl/video/IGPUCommandPool.h

```diff
@@ -31,13 +66,219 @@ class IGPUCommandPool : public core::IReferenceCounted, public IBackendObject
         // Vulkan: const VkCommandPool*
         virtual const void* getNativeHandle() const = 0;
 
+        core::CMemoryPool<core::GeneralpurposeAddressAllocator<uint32_t>, core::default_aligned_allocator, false, uint32_t> m_commandSegmentPool;
```

Comment:
PoolAddressAllocator, no need for generalpurpose because blocks are fixed size

Reply:
also this CANNOT be public!

Reply:
Resolved.

Reply:
Still need to make it `PoolAddressAllocator`.

Reply:
Resolved.